src/frontend/admin/src/components/templates/organizations/form/OrganizationForm.tsx

```diff
@@ -98,6 +99,22 @@ export function OrganizationForm(props: Props) {
     }
   };
 
+  const getUploadedSignature = (): ThumbnailDetailField[] => {
```

Comment:
Is the org signature still needed now that Lex Persona will handle it?

Reply:
For the moment, we need it